docker/deployment/build_context.sh

```diff
@@ -32,6 +32,11 @@ cargo install \
     --tag "v$oasis_core_version" \
     oasis-core-tools
 
+
+# XXX: needed while we use a different rust toolchain than the one provided in
+# the oasis-core ci image. Remove once we are able to bump the toolchain.
+rustup target add x86_64-fortanix-unknown-sgx
```

Comment:
on some level, I think that we want to template these files. not worth the effort, though.